### DISCUSSION TITLE
I need to add description and other param also in JsonMaker.py

### DISCUSSION THREAD

**Discussion by rohit777anand** [rohan472000/SnippetHub#4] [Q&A] [answered]
what will be the changes in code if i need to add extra parameters like description, because in  your code it was common and pre-written for all???

**Comment by rohan472000** [comment, marked as answer]
In case you need to add any parameters , first you should define that in cloumn.txt tables for every column in a row(separately), like-

`name, type, desc, age ----> tom, male, student, 23`

and then add like this `desc = values[2]` and `age = values[3]`

**Reply by rohit777anand** [reply]
oh ok..thanks
